terraform/moe_user_write.tf

```diff
@@ -0,0 +1,39 @@
+resource "aws_iam_user" "create_moe_user_write" {
+  name = "moe_user_write"
+  path = "/system/"
+}
+
+resource "aws_iam_access_key" "create_moe_user_write" {
+  user = "${aws_iam_user.create_moe_user_write.name}"
+}
+
+resource "aws_iam_user_policy" "create_moe_user_write_role" {
+  name = "moe_bucket_write_access"
+  user = "${aws_iam_user.create_moe_user_write.name}"
+
+  policy = <<EOF
+{
+    "Version": "2012-10-17",
+    "Statement": [
```

Comment:
These permissions are identical to those of the read user.  This needs to be fixed (maybe `GetObject` should be `PutObject`) or one of these users should be removed as it is unnecessary.